docs/docs/icechunk-python/parallel.md

```diff
@@ -59,9 +59,9 @@ def write_timestamp(*, itime: int, session: Session) -> None:
 
 Now execute the writes.
 
-```python exec="on" session="parallel" source="material-block" result="code"
+<!-- ```python exec="on" session="parallel" source="material-block" result="code" -->
+```python
```

Comment:
this example runs fine for me locally, but ends up not writing anything to store when running on readthedocs.

Reply:
OK that is concerning. Does it work when you build the docs locally too?

Reply:
yes, `mkdocs build` runs fine for me. I can push a version with this reactivated so we can lookat the error.